kodilla-hibernate/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_course?serverTimezone=Europe/Warsaw&useSSL=False&allowPublicKeyRetrieval=true
+spring.datasource.username=kodilla_user
+spring.datasource.password=kodilla_password
+
+spring.jpa.database=mysql
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
tu masz powiedziane, ze ma byc zmieniony schemat bazy danych, jesli cokolwiek sie w nim zmieni. Tak nie robimy w projektach na codzien .o jest to, o co pytalas :)